R/plumber-step.R

```diff
@@ -6,8 +8,11 @@
 #' itself and no other filters or endpoints should be evaluated for this
 #' request.
 #' @export
-forward <- function(){
-  .globals$forwarded <- TRUE
+forward <- function() {
+  structure(forward_class, class = forward_class)
```

Comment:
This is changing the semantics of forward, isn't it? Previously you just had to call forward() somewhere in your filter but now you have to make sure that forward() is what's returned?

Reply:
Correct.  I've noted this in breaking changes within the NEWS.  All examples show it as a final step, so I'm hoping this will be minimal impact.

There probably needs to be more documentation around this.

Reply:
Oh, can't we just make it work?

Reply:
I worry that'd be a notable breaking change. Might require a major version bump if we can't get it to work IMHO.

Reply:
fixed!